compiler/il/OMRBlock.cpp

```diff
@@ -1143,6 +1143,95 @@ OMR::Block::createConditionalBlocksBeforeTree(TR::TreeTop * tree,
    return remainderBlock;
    }
 
+/**
+ * calling this function on block_A will convert
```

Comment:
Could you add a note explaining that block_A is assumed to only have one successor (and maybe why this is the case)? For example `calling this function on block_A, which must have only one successor, will convert`.

Although the assumption is mentioned later on, I think that calling it out explicitly earlier on makes it easier to understand.